### PR TITLE
Update transportation documents field to be transportation or other docs

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#3898

screen shot 2017-07-18 at 12 22 41 pm

screen shot 2017-07-18 at 12 23 24 pm

### CODE REVIEW

**Comment by stacyhlinn** [comment]
LGTM. Wanted to confirm that the upload is not required if transportation reimbursement is not claimed.

File: src/js/burials/config/form.js [hunk 1]
```diff
@@ -523,12 +530,18 @@ const formConfig = {
             }), {
               'ui:required': form => form.burialAllowanceRequested === 'service',
             }),
-            transportationReceipts: _.assign(fileUploadUI('Documentation for transportation of the Veteran’s remains', {
+            transportationReceipts: _.assign(fileUploadUI('Documentation for transportation of the Veteran’s remains or other supporting evidence', {
               addAnotherLabel: 'Add Another Document',
               hideIf: form => _.get('view:claimedBenefits.transportation', form) !== true
             }), {
               'ui:required': form => _.get('view:claimedBenefits.transportation', form) === true,
```
Comment (cehsu) [inline src/js/burials/config/form.js]: Thanks @stacyhlinn, the logic for conditionally requiring document upload was already in place.

**Comment by jbalboni** [comment]
@cehsu You'll also need to remove the `depends` property on the doc upload page. We want this page to always show up.

**Comment by cehsu** [comment]
Thanks @jbalboni, change made. I also removed the hide logic from the transportation receipt field. Should this still be hidden unless certain conditions are met (either transportation benefits is selected, or no service periods are provided?)?

**Comment by jbalboni** [comment]
@cehsu No, that field should always be visible, since it's now for transportation receipts and anything else.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**